dev/PagerControl/PagerControl.xaml

```diff
@@ -115,81 +134,84 @@
+                                    Style="{TemplateBinding PreviousButtonStyle}"
                                     FontFamily="{StaticResource SymbolThemeFontFamily}"
                                     Content="&#xE76B;"/>
-                            <TextBlock x:Name="PreviousPageButtonText" VerticalAlignment="Center" HorizontalAlignment="Center"
-                                        Text="{TemplateBinding PreviousPageButtonText}"
-                                        Visibility="{Binding ElementName=PreviousPageButton, Path=Visibility}"/>
                         </StackPanel>
-                        <StackPanel x:Name="BoxPanels" Visibility="Collapsed" Orientation="Horizontal" Grid.Row="0" Grid.Column="1">
-                            <TextBlock VerticalAlignment="Center" Margin="0,0,5,0" Text="{TemplateBinding PrefixText}" />
+                        
+                        <StackPanel x:Name="BoxPanels" Visibility="Collapsed" Orientation="Horizontal"
+                                Grid.Column="1">
+                            <TextBlock x:Name="PrefixTextLabel" Text="{TemplateBinding PrefixText}" VerticalAlignment="Center" Margin="0,0,5,0"/>
                             <!--NumberBox Display-->
-                            <controls:NumberBox x:Name="NumberBoxDisplay" HorizontalAlignment="Center" VerticalAlignment="Center"
-                                            Maximum="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=NumberOfPages}" Minimum="1"
-                                            Value="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=SelectedIndex, Mode=TwoWay}"
-                                            SmallChange="1" LargeChange="5"
-                                            IsEnabled="False" Visibility="Collapsed" />
+                            <muxc:NumberBox x:Name="NumberBoxDisplay" HorizontalAlignment="Center" VerticalAlignment="Center"
+                                    Margin="5,0,0,0"
+                                    Maximum="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=NumberOfPages, Mode=OneWay}"
+                                    Minimum="1" SmallChange="1" LargeChange="5" Visibility="Collapsed" />
                             <!--ComboBox Display-->
                             <ComboBox x:Name="ComboBoxDisplay" HorizontalAlignment="Center" VerticalAlignment="Center"
                                     ItemsSource="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=TemplateSettings.Pages}"
-                                    IsEnabled="False" Visibility="Collapsed"/>
-                            <TextBlock VerticalAlignment="Center" Margin="5,0,5,0" Text="/"/>
-                            <TextBlock VerticalAlignment="Center" Margin="0,0,5,0" Text="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=NumberOfPages}"/>
+                                    Visibility="Collapsed" SelectedIndex="0"/>
```

Comment:
>SelectedIndex [](start = 59, length = 13)

Selected index defaults to 0 right?

Reply:
ComboBox.SelectedIndex defaults to -1, PagerControl.SelectedPageIndex defaults to 0 though, so we need to explicitly set it here.